### ISSUE TITLE
Settings screen - Sub menu - Accessibility issues

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#431]
Related to #239 

Settings screen
1. When a language is selected it should read as "name Language selected " button. For instance if the English language is selected it should read as "English language selected "
2. The other unselected option should be read as "French language" button and vice-versa.

**Comment by Rajat-Sen** [comment]
Validated!
